tests/checksum/benchmarks_json/gaussian_linear_wake.SI.1Rank.json

```diff
@@ -1,22 +1,22 @@
 {
   "lev=0": {
-    "Bx": 2235.8904092752,
```

Comment:
Could you raise the tolerance for the CPU checksum on this test by 1 OOM, and see if this passes then? This will still be tighter tolerance than before #906.